tests/createProjectDriver.py

```diff
@@ -0,0 +1,136 @@
+
+from loginDriver import logIn
+from selenium.webdriver import Chrome
+import time
+
+class createProject:
+
+
+    def __init__(self):
+        self.driver = Chrome()
+    
+    def Close(self):
+        self.driver.quit()
+    
+    def setProjectName(self, projectname, projectpassword):
+        #This is to set up the projectName + description
+        self.driver.find_element_by_id("project_name").send_keys(projectname)
+        self.driver.find_element_by_id("project_description").send_keys(projectpassword)
+    
+    def download(self):
+        self.driver.find_element_by_link_text("(Download a sample roster files)").click()
+        self.driver.implicitly_wait(10)
+    
+    def setRoster(self, studentFile):
+
+        self.driver.find_element_by_id("student_file").send_keys(studentFile)
+    
+    def setRubrics(self, jsonFile):
+        self.driver.find_element_by_id("json_file").send_keys(jsonFile)
+    
+    def driver_createProject(self, username, password, projectname, projectpassword, 
+        studentFile = "C:/Users/Wangj/Downloads/sample_roster.xlsx", jsonFile = "C:/Users/Wangj/Downloads/teamwork_scale3.json"):
```

Comment:
Why are we using selenium to download these files to the Downloads folder? Why not just use the copies that are already in the local repository?

Reply:
The current form of this driver file utilizes the files in the local repo.